src/napari_manual_transforms/__init__.py

```diff
@@ -6,6 +6,6 @@
     # package is not installed
     __version__ = "uninstalled"
 
-from ._widget import RotationWidget
+from ._widget import TransformationWidget
 
-__all__ = ["RotationWidget"]
+__all__ = ["TransformationWidget"]
```

Comment:
I don't imagine anyone was directly importing this widget... but just in case, we need to deprecate the name before removing it

```suggestion
__all__ = ["TransformationWidget"]

def __getattr__(name: str) -> object:
    if name == "RotationWidget":
        from warnings import warn
        warnings.warn(
            "'RotationWidget' has been renamed to 'TransformationWidget'.",
            DeprecationWarning,
            stacklevel=2,
        )
        return TransformationWidget
    raise AttributeError(f"module {__name__} has no attribute {name}")
```

Reply:
I have implemented your suggestion.